src/components/Inventory/view/BatchDetailsModal.tsx

```diff
@@ -193,7 +193,7 @@ export default function BatchDetailsModal(props: BatchDetailsModalProps): JSX.El
                 <Textfield
                   id='seedilingBatch'
                   value={record.batchNumber}
-                  onChange={onChange}
+                  onChange={(value) => onChange('seedilingBatch', value)}
```

Comment:
typo `seedlingsBatch` .. guessing we don't need the onChange here and in the other places - since it's readonly?